tdei-ui/src/routes/UploadDataset/UploadDataset.js

```diff
@@ -60,7 +60,9 @@ const UploadDataset = () => {
 
   const handleClose = () => {
     setToast(false);
-    setCurrentStep(2); // reverting to metadata step.
+    if(errorMessage !== "Derived dataset id not found"){
+      setCurrentStep(2); // reverting to metadata step.
+    }
     console.log(currentStep)
```

Comment:
remove console logs for data logging everywhere

Reply:
Removed console logs